pkg/aws/ec2_test.go

```diff
@@ -730,3 +731,235 @@ func TestGetEC2SecurityGroupByNameE(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, expectedOutput, actualOutput)
 }
+
+func TestAssertEC2VolumeType_MatchWithGP2(t *testing.T) {
+	// Setup
+	instanceID := "i546acas321sd"
+	volumeId := "v123dfasd92"
+	deviceName := "/dev/sdc"
+	kmsKeyID := "/key/id"
+	encrypted := true
+	volumeType := types.VolumeTypeGp2
+	volumeIops := aws.Int32(0)
+	volumeThroughput := aws.Int32(0)
```

Comment:
**Suggestion:** Would you please use the [SDK v2 version of this method](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2@v1.13.0/aws#Int32)?

Reply:
addressed